jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -246,7 +206,6 @@ node {
         cleanWs()
         dir('installation') {
             checkoutGitRepo(installationGitUrl, installationGitRef, githubCredentialsID)
-            ensureEvalsDir()
```

Comment:
Would there be any issues if we removed this? @mikenairn

Reply:
It probably isn't actually required in here as long as we are never dealing with any branch other than master i.e v1.2. I can't see that ever happening, so should be OK to remove now.